### PR TITLE
added some gateway interfaces

### PR DESCRIPTION
**Pull request by oblomov-dev**
this solves the issues here https://github.com/abap2UI5-connectors/fiori-launchpad-kpi-onpremise/blob/2812020ed2cb8a14e964eaab05ad0b7fe0927789/abaplint.jsonc#L10

### CODE REVIEW

**Comment by larshp** [comment]
suggest `git clone` ing it, and open in vscode

https://marketplace.visualstudio.com/items?itemName=larshp.standalone-abap-development

File: src/gateway/#iwbep#if_mgw_appl_srv_runtime.intf.abap [hunk 1]
```diff
@@ -38,4 +38,21 @@ INTERFACE /iwbep/if_mgw_appl_srv_runtime PUBLIC.
...
\ No newline at end of file
+  METHODS get_entityset
+    IMPORTING
+      !iv_entity_name           TYPE string OPTIONAL
+      !iv_entity_set_name       TYPE string OPTIONAL
+      !iv_source_name           TYPE string OPTIONAL
+      !it_filter_select_options TYPE string OPTIONAL
+      !it_order                 TYPE string OPTIONAL
+      !is_paging                TYPE string OPTIONAL
+      !it_navigation_path       TYPE stringoptional
```
Comment (larshp) [inline src/gateway/#iwbep#if_mgw_appl_srv_runtime.intf.abap:49]:
```suggestion
      !it_navigation_path       TYPE string OPTIONAL
```

**Comment by oblomov-dev** [comment]
thanks for the hint with the abap-dev extension, big help with auto highlighting of abaplint issues 👍

just can not find the option for the auto fix functionality so i still install the cli for this and do:
npm install @abaplint/cli -g
abaplint --fix

if i missed a quicker option directly via UI, let me know.

**Comment by larshp** [comment]
I also recommend using https://marketplace.visualstudio.com/items?itemName=usernamehw.errorlens, but its personal preference

thanks

**Pull request merged**